### DISCUSSION TITLE
jira_transition_to: boolean or string ?

### DISCUSSION THREAD

**Discussion by toxisch** [jertel/elastalert2#719] [Q&A] [answered]
Hi, 
in Elastalert2 docs the config property  "jira_transition_to" is described as a string value. But in schema it is validated as a boolean. The elastalert won't start, if I configure a string.

is this a bug?

Doc:
jira_transition_to: If jira_bump_tickets is true, Transition this ticket to the given Status when bumping. Must match the text of your Jira implementation’s Status field.
Example usage:
jira_transition_to: 'Fixed'

Schema:
https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml
jira_transition_to: {type: boolean}

**Comment by nsano-rururu** [comment, marked as answer]
>Hi,
>in Elastalert2 docs the config property "jira_transition_to" is described as a string value. But in schema it is validated as a boolean. The >elastalert won't start, if I configure a string.
>
>is this a bug?

If you think it's a bug, please make a pull request.

**Reply by nsano-rururu** [reply]
It's strange that the current implementation sets boolean False by default.
https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/jira.py#L89

**Reply by nsano-rururu** [reply]
You can tell if the default value is an empty string or None. So if the s schema yam data type is string. I wonder if that should be the case

**Reply by toxisch** [reply]
The value (False) is compared with all available transition names (strings) of jira:
https://github.com/jertel/elastalert2/blob/d6c8c8934ad7cdbbf3f8ad018593b7951790a954/elastalert/alerters/jira.py#L279

If there is no match, no transion is processed. If I compare a Boolean with a String in Python it is always a False. So there will be never a match. 

I think value type shoud be String
and value default should be None

**Reply by nsano-rururu** [reply]
@toxisch 

Please make a pull request.
